presto-client/src/main/java/com/facebook/presto/client/OkHttpUtil.java

```diff
@@ -178,7 +180,7 @@ public static void setupSsl(
             // load TrustStore if configured, otherwise use KeyStore
             KeyStore trustStore = keyStore;
             if (trustStorePath.isPresent()) {
-                trustStore = loadTrustStore(new File(trustStorePath.get()), trustStorePassword);
+                trustStore = loadTrustStore(new File(trustStorePath.get()), trustStorePassword, trustStoreType.get());
```

Comment:
Please add a validation to ensure that these parameters are present before getting.

Reply:
We have set default values in ClientOptions.java

<img width="718" alt="image" src="https://github.com/prestodb/presto/assets/148428873/3c876f93-d973-4f6a-8249-a367db22719e">


<img width="653" alt="image" src="https://github.com/prestodb/presto/assets/148428873/a3679b26-dba9-49ff-ab33-3377d147f9e5">

Reply:
Understood, but that's in a different module.  Best practice is to either fail, or only pass in String (and check not null).